app/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,48 @@
+<resources>
+    <string name="app_name">Синий спидометр</string>
+
+    <string name="permission_location_no">Не предоставлено разрешение на местоположение</string>
+    <string name="permission_location_coarse">Разрешено только приблизительное местоположение</string>
+    <string name="permission_location_fine">Разрешено точное местоположение</string>
+    <string name="permission_location_grant_prompt">Предоставить разрешение на местоположение</string>
+    <string name="permission_location_description">Это приложение использует спидометр на основе службы определения местоположения устройства, поэтому необходимо разрешение на точное определение местоположения.</string>
+
+    <string name="error_soft_location_not_enabled">Система местоположения сейчас не включена.</string>
+    <string name="error_soft_location_provider_not_enabled">Поставщик местоположения \"%1$s\" сейчас не включён.</string>
+
+    <string name="menu_this_app_digital_speedometer_fullscreen">Цифровой спидометр (полноэкранный)</string>
+    <string name="menu_this_app_licensing_information">Информация про лицензии</string>
+
+    <string name="metrics_speed">Скорость</string>
+    <string name="metrics_altitude">Высота</string>
+    <string name="metrics_msl_altitude">Высота MSL</string>
+    <string name="metrics_wgs84_altitude">Высота WGS84</string>
+    <string name="metrics_accuracy">Точность</string>
+    <string name="metrics_msl_altitude_accuracy">Точность высоты MSL</string>
+    <string name="metrics_info_time">Время</string>
+
+    <string name="unit_km_per_hour">км/ч</string>
+    <string name="unit_knot">узлов</string>
+    <string name="unit_meter_per_second">м/с</string>
+    <string name="unit_mile_per_hour">миль/ч</string>
+    <string name="unit_meters">м</string>
+    <string name="unit_feet">фут</string>
+    <string name="unit_angle_deg">%1$d°%2$02d\'%3$02d.%4$01d\"</string>
+
+    <string name="dialog_select_location_provider">Выберите поставщика местоположения</string>
+    <string name="dialog_select_speed_unit">Выберите единицы измерения скорости</string>
+    <string name="dialog_select_altitude_unit">Выберите единицы измерения высоты</string>
+
+    <string name="config_location_provider">Поставщик местоположения</string>
+    <string name="config_speed_unit">Единицы скорости</string>
+    <string name="config_altitude_unit">Единицы высоты</string>
+
+    <string name="coordinate_display_north">С</string>
+    <string name="coordinate_display_south">Ю</string>
+    <string name="coordinate_display_east">В</string>
+    <string name="coordinate_display_west">З</string>
```

Comment:
I noticed that it shows like this.
![スクリーンショット 2025-06-05 004145](https://github.com/user-attachments/assets/0dcf36ff-94b2-4f42-86cd-7fa0b640a8d3)

In my eye it looks like `3122°05'02.4"` not `З122°05'02.4"`.

If it is fine to Russian speakers it is fine.

But if it would be confusing even for Russian speakers, I suggest the following options:
- Again, just keeping as current code if it is not confusing to Russian speakers
- Just keeping N, E, S, and W
- adding `\00a0` just after the alphabet like `<string name="coordinate_display_west">З\u00a0</string>`, which looks like the following:
![スクリーンショット 2025-06-05 005007](https://github.com/user-attachments/assets/d9144c0b-244c-43db-8203-735bf8a2d9be)

- Changing here